modules/signatures.js

```diff
@@ -99,27 +132,7 @@ Signatures.prototype.shared = {
 		return modules.transport.shared.postSignatures(
 			{ signatures },
 			(err, res) => {
-				var processingError = /(error|processing)/gi;
-				var badRequestBodyError = /(invalid|signature)/gi;
-
-				if (res.success === false) {
-					if (processingError.exec(res.message).length === 2) {
-						return setImmediate(
-							cb,
-							new ApiError(res.message, apiCodes.PROCESSING_ERROR)
-						);
-					} else if (badRequestBodyError.exec(res.message).length === 2) {
-						return setImmediate(
-							cb,
-							new ApiError(res.message, apiCodes.BAD_REQUEST)
-						);
-					}
-					return setImmediate(
-						cb,
-						new ApiError(res.message, apiCodes.INTERNAL_SERVER_ERROR)
-					);
-				}
-				return setImmediate(cb, null, { status: 'Signature Accepted' });
+				__private.processPostResult(res, cb);
```

Comment:
Same here.

Reply:
I just moved the response logic to its own function (for reuse), the error wasn't been handled before. Good point, I'll look into this.